projects/gloo/pkg/plugins/tracing/plugin_test.go

```diff
@@ -601,167 +598,6 @@ var _ = Describe("Plugin", func() {
 			})
 		})
 
-		Describe("when opencensus provider config", func() {
-			It("translates the plugin correctly using OcagentAddress", func() {
```

Comment:
Did we intentionally delete this test?

Reply:
Yes. Open Census support has been deprecated in Envoy for a long time, and they finally removed the protos from go-control-plane.